pkg/ccl/backupccl/show.go

```diff
@@ -863,14 +871,60 @@ func backupShowerFileSetup(inCol tree.Expr) backupShower {
 	}
 }
 
+// getManifestDirs gathers the path to the directory of each backup manifest,
+// relative to the collection root. Consider the following example: Suppose a
+// backup chain contains a full backup with a defaultURI of
+// 'userfile:///foo/fullSubdir' and an incremental backup with a defaultURI of
+// 'userfile:///foo/incrementals/fullSubdir/incrementalSubdir'. getManifestDirs
+// would return a relative path to the full backup manifest's directory, '/fullSubdir', and
+// to the incremental backup manifest's directory
+// '/incrementals/fullSubdir/incrementalSubdir'.
+func getManifestDirs(fullSubdir string, defaultUris []string) ([]string, error) {
+	manifestDirs := make([]string, len(defaultUris))
+
+	// The full backup manifest path is always in the fullSubdir
+	manifestDirs[0] = fullSubdir
+	if len(strings.Split(defaultUris[0], fullSubdir)) != 2 {
+		return nil, errors.Newf("the full backup URI %s does not contain 1 instance of the subdir %s"+
+			"", defaultUris[0], fullSubdir)
+	}
+	fullCollectionRoot := strings.Split(defaultUris[0], fullSubdir)[0]
+
+	if len(defaultUris) == 1 {
+		return manifestDirs, nil
+	}
+	// unless the incremental backup is stored in the default incremental
+	// directory (i.e. fullCollection/incrementals/fullSubdir), the incremental
+	// chain is stored in collection/fullSubdir. Note: for incremental backups
+	// created with incremental_location, the collection path will be different
+	// than the full backup's.
+	incSubdir := fullSubdir
+	if len(strings.Split(defaultUris[1], fullSubdir)) != 2 {
+		return nil, errors.Newf("the incremental backup URI %s does not contain 1 instance of the"+
+			" subdir %s", defaultUris[1], fullSubdir)
+	}
+	incLocNoSubdir := strings.Split(defaultUris[1], fullSubdir)[0]
+
+	if path.Join(fullCollectionRoot, DefaultIncrementalsSubdir) == incLocNoSubdir {
+		// incremental backups stored in default incremental directory
```

Comment:
nit: period at the end of comment, and captialization of first letter.

Reply:
done.